wandb/sdk/wandb_init.py

```diff
@@ -515,6 +515,7 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
 
         manager = self._wl._get_manager()
         if manager:
+            logger.info(f"setting up manager: {manager}")
```

Comment:
manager object isnt very printable.. it looks like:
`<wandb.sdk.wandb_manager._Manager object at 0x12add5340>`
maybe remove manager or the log altogether?
```suggestion
            logger.info(f"setting up manager")
```